Jenkinsfile

```diff
@@ -181,28 +184,84 @@ pipeline {
           }
         }
         stage('Release') {
-          options { skipDefaultCheckout() }
+          options {
+            skipDefaultCheckout()
+            timeout(time: 12, unit: 'HOURS')
+          }
+          environment {
+            HOME = "${env.WORKSPACE}"
+          }
           when {
             beforeAgent true
-            tag pattern: '@elastic/apm-rum@\\d+\\.\\d+\\.\\d+$', comparator: 'REGEXP'
+            beforeInput true
+            allOf {
+              branch 'master'
+              expression { return params.release }
+            }
           }
           stages {
-            stage('Opbeans') {
-              environment {
-                REPO_NAME = "${OPBEANS_REPO}"
+            stage('Notify') {
+              options { skipDefaultCheckout() }
+              steps {
+                deleteDir()
+                unstash 'source'
+                unstash 'cache'
+                dir("${BASE_DIR}") {
+                  prepareRelease() {
+                    script {
+                      sh(label: 'Lerna version dry-run', script: 'lerna version --no-push --yes', returnStdout: true)
+                      def releaseVersions = sh(label: 'Gather versions from last commit', script: 'git log -1 --format="%b"', returnStdout: true)
+                      log(level: 'INFO', text: "Versions: ${releaseVersions}")
+                      emailext subject: "[${env.REPO}] Release ready to be pushed", to: "${NOTIFY_TO}",
```

Comment:
an email notification with the link to the input approval and the list of packages that will be bumped